security/oc_tls.c

```diff
@@ -1368,6 +1431,11 @@ static int
 oc_tls_load_identity_cert_chain(mbedtls_ssl_config *conf, size_t device,
                                 int credid)
 {
+  if (credid < -1) {
```

Comment:
for `oc_tls_load_identity_cert_chain` is -1 OK?

Reply:
added define for ANY. Yes, any is a default behavior.